Dockerfile.alpine

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12-alpine3.20
```

Comment:
even with alpine this image is `1.25GB`, probably multi-stage builds can help to reduce the size